.browserslistrc

```diff
@@ -1,4 +1,4 @@
 # Browsers that we support
 
 > 0.5%, last 2 versions, Firefox ESR,
-not dead, not op_mini all, not KaiOS < 4.0
+not dead, not op_mini all, not KaiOS < 4.0, not and_qq > 0, not and_uc > 0
```

Comment:
Just out of curiosity, what does "and_qq" and "and_uc" mean?

Reply:
that's excluding 
UC Browser for Android
and QQ Browser

otherwise we get lots of more warnings .. 

https://browsersl.ist/#q=%3E+0.5%25%2C+last+2+versions%2C+Firefox+ESR%2C%0Anot+dead%2C+not+op_mini+all%2C+not+KaiOS+%3C+4.0%2C+not+and_qq+%3E+0%2C+not+and_uc+%3E+0